gateway-messages/src/sp_to_mgs.rs

```diff
@@ -106,6 +106,9 @@ pub enum SpResponse {
     CabooseValue,
 
     SerialConsoleKeepAliveAck,
+    ResetComponentPrepareAck,
+    ResetComponentTriggerAck,
+    SwitchDefaultImageAck,
```

Comment:
Please add coverage for these to `gateway_messages::tests::versioning::v2::sp_response()`